modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceItem.js

```diff
@@ -313,6 +320,7 @@ ExperienceItem.propTypes = {
 	lockedDecreasePriority: PropTypes.bool.isRequired,
 	lockedIncreasePriority: PropTypes.bool.isRequired,
 	onDeleteExperience: PropTypes.func.isRequired,
+	onDuplicateExperience: PropTypes.func.isRequired,
```

Comment:
This is so nice! If you use `PropTypes` you will be notices in the tests if some properties are missing on your tests :) Also, helps to better understand the component. Well done!